go.mod

```diff
@@ -1,4 +1,4 @@
-module zinx
+module github.com/aceld/zinx
```

Comment:
Hi, please try "go mod tidy" and fix all warnings in this package.

Reply:
when i require github.com/aceld/zinx@latest in go.mod, it will also attend a warning , parsing go.mod: unexpected module path "zinx" , cause of go.mod file module zinx has been set. when i download zinx from github by myself, i can move it to gopath or vendor path. so i hope the module name could be set to github.com/aceld/zinx.

Reply:
yes, it's shoul be github.com/aceld/zinx.

Reply:
> when i require github.com/aceld/zinx@latest in go.mod, it will also attend a warning , parsing go.mod: unexpected module path "zinx" , cause of go.mod file module zinx has been set. when i download zinx from github by myself, i can move it to gopath or vendor path. so i hope the module name could be set to github.com/aceld/zinx.

updated, please check it.